tests/bsim/bluetooth/host/att/timeout/prj.conf

```diff
@@ -0,0 +1,25 @@
+CONFIG_ASSERT=y
```

Comment:
This seems to be the exact same configuration as tests/bsim/bluetooth/host/att/long_read/prj.conf
Note that for most tests we spend more time building them than actually running them.
If the .conf is the same, is there any reason to not build these as two tests that are part of the same image?

Reply:
I am fully agree with you. But the pattern has already been defined. I also see the duplicate code has been used in other tests. But I think it is a separate task to get rid of duplicate code and merge tests into a single image. I saw there was some work done towards this, but seems was not finished.

@alwa-nordic, what do you think? Do you have a task in your backlog for this?

Reply:
Let's leave it out of scope of this PR and create an issue for it. I started some work on a combined build a while ago: https://github.com/zephyrproject-rtos/zephyr/pull/77122, and I would like it done.

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/79512